script/test_k8s/kubernetes.sh

```diff
@@ -103,6 +103,19 @@ test_k8s() {
     sleep 2 # Sleep for k8s to catch up to deployment
     echo -e "\n${RED}kompose down -f $f ${NC}\n"
     ./kompose down -f $f
+    echo -e "\nTesting controller=daemonset key\n"
```

Comment:
I think we should find an alternative way to write these integration tests (perhaps in a separate function? and only running the httpd.yaml test?) or even a separate file...

It's a bit messy at the moment and we really need to refactor everything (using `e2e_test.go` similar to our other project we're working on.

Do you mind opening an issue explaining that we should refactor our integration tests to use a Go file / make this better? As well as add a comment (and spacing above this line) with a `# TODO: need to reorganize / refactor`
 Otherwise, this LGTM (for now).

Reply:
Thanks for adding issue as well.
Ready to merge ?